Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -179,7 +179,11 @@ extension BrowserViewController: WKNavigationDelegate {
     // First special case are some schemes that are about Calling. We prompt the user to confirm this action. This
     // gives us the exact same behaviour as Safari.
     if ["sms", "tel", "facetime", "facetime-audio"].contains(url.scheme) {
-      handleExternalURL(url, isMainFrame: navigationAction.targetFrame?.isMainFrame == true)
+      
+      // Do not allow opening external URLs from child tabs
+      if (webView as? TabWebView)?.tab?.parent == nil {
+        handleExternalURL(url, isMainFrame: navigationAction.targetFrame?.isMainFrame == true)
```

Comment:
Since it is only used `WKNavigationDelegate`, maybe we can pass the tab instance to `handleExternalURL` function and do not add the check couple of times.

Reply:
Done.